src/action_cc.cc

```diff
@@ -375,13 +375,27 @@ NM_ACTION_(power) {
 
 NM_ACTION_(cmd_spawn) {
     #define NM_ERR_RET nullptr
+
     char *tmp = strdup(arg);
+
     char *cmd = tmp;
+    char *tmp1 = strtrim(strsep(&cmd, ":"));
     bool quiet = false;
-    if (!strncmp(tmp, "quiet:", 6)) {
-        cmd += 6;
+    if (cmd && !strncmp(tmp1, "quiet", 5)) {
```

Comment:
This doesn't work if it is something like `quiet_`.

Reply:
True, once it's trimmed, a simple `strcmp` make smuch more sense ;).